src/Bicep.Local.Extension/Rpc/BicepExtensionImpl.cs

```diff
@@ -5,6 +5,7 @@
 using System.Text.Json.Nodes;
 using Bicep.Local.Extension.Protocol;
 using Grpc.Core;
+using Grpc.Net.Client;
```

Comment:
Do you know why this was necessary?

Reply:
No clue, might be VS adding references when performing autocompletion, I just removed the unnecessary using statements btw.